code/game/machinery/cloning.dm

```diff
@@ -381,10 +370,47 @@
 
 /obj/machinery/clonepod/deconstruct(disassembled = TRUE)
 	if(occupant)
-		locked = FALSE
 		go_out()
 	..()
 
+/obj/machinery/clonepod/proc/maim_clone(mob/living/carbon/human/H)
+	unattached_flesh.Cut()
+
+	H.setCloneLoss(CLONE_INITIAL_DAMAGE)     //Yeah, clones start with very low health, not with random, because why would they start with random health
+	H.setBrainLoss(CLONE_INITIAL_DAMAGE)
+	// In addition to being cellularly damaged and having barely any
+	// brain function, they also have no limbs or internal organs.
+	var/list/zones = list("r_arm", "l_arm", "r_leg", "l_leg")
+	for(var/zone in zones)
+		var/obj/item/bodypart/BP = H.get_bodypart(zone)
+		BP.drop_limb()
+		BP.forceMove(src)
+		unattached_flesh += BP
+
+	var/list/organs = list(
+		H.getorganslot("heart"),
+		H.getorganslot("lungs"),
+		H.getorganslot("appendix"),
+		H.getorganslot("tongue")
+	)
+	for(var/o in organs)
+		var/obj/item/organ/organ = o
+		if(!istype(organ))
+			continue
+		organ.Remove(H)
+		organ.forceMove(src)
+		unattached_flesh += organ
+
+	shuffle_inplace(unattached_flesh)
+	flesh_number = unattached_flesh.len

```

Comment:
unattached_flesh is pick_n_take()'d so its len goes down. Need to remember how big it started.